3rdparty/carotene/hal/tegra_hal.hpp

```diff
@@ -1433,15 +1433,16 @@ inline int TEGRA_MORPHFREE(cvhalFilter2D *context)
 
 #define TEGRA_RESIZE(src_type, src_data, src_step, src_width, src_height, dst_data, dst_step, dst_width, dst_height, inv_scale_x, inv_scale_y, interpolation) \
 ( \
-    interpolation == CV_HAL_INTER_LINEAR ? \
+    /*nearest neighbour interpolation disabled due to rounding accuracy issues*/ \
+    /*interpolation == CV_HAL_INTER_LINEAR ? \
```

Comment:
> nearest neighbour
> CV_HAL_INTER_LINEAR

Looks like this comment doesn't match to actual code.

@terfendail Does "bit-exact" changes affects this resize mode? or it should be a separate mode?

Reply:
"bit-exact" linear resize is implemented as separate mode `INTER_LINEAR_EXACT = 5` so it shouldn't affect `CV_HAL_INTER_LINEAR = 1` at least directly. However "bit-exact" change affect the mode used for internal resizes almost everywhere so it could affect the reference

Reply:
Thanks @alalek , I updated the comment.

@terfendail , I understand that this doesn't need to be a bit-exact, so loosening the threshold is another choice (keep the implementation as is)

May be it's a choice to keep the current implementation since we have a "bit-exact" implementation.
How does that sound ?

Reply:
I think that it would be better to retain threshold as low as possible even for "bit-inexact" resize. So I also prefer to disable carotene for linear resize while the difference root cause is unclear.

Reply:
Ii have a feeling that it's a rounding error.
Not the rounding of pixel value it self, but the pixel coordinate.

Bi-linear interpolation picks up 4 pixels, and computes a weighted average.  If we pick from different pixel due to the rounding difference of coordinate, the result pixel value could be differ more than 1.
That explains why the difference goes over threshold, and also why it happens rarely.


It might be worth of taking a different way of evaluation, such as number of pixels which goes over the threshold.